docs/content/tutorials/kic/deploy-with-kic.md

```diff
@@ -395,48 +327,48 @@ The example below uses the snippets annotation. Starting with NGINX Plus Ingress
 
-1. Run the NGINX Plus Ingress Controller with both the `-enable-prometheus-metrics` and `-enable-latency-metrics` command line arguments.
-    The NGINX Plus Ingress Controller exposes [NGINX Plus metrics](https://github.com/nginxinc/nginx-prometheus-exporter#exported-metrics) and latency metrics
+1. Run the NGINX Ingress Controller with both the `-enable-prometheus-metrics` and `-enable-latency-metrics` command line arguments.
+    The NGINX Ingress Controller exposes [NGINX metrics](https://github.com/nginxinc/nginx-prometheus-exporter#exported-metrics) and latency metrics
     in Prometheus format via the `/metrics` path on port 9113. This port is customizable via the `-prometheus-metrics-listen-port` command-line argument; consult the
-    [Command Line Arguments](https://docs.nginx.com/nginx-ingress-controller/configuration/global-configuration/command-line-arguments/) section of the NGINX Plus Ingress Controller docs for more information on available command line arguments.
+    [Command Line Arguments](https://docs.nginx.com/nginx-ingress-controller/configuration/global-configuration/command-line-arguments/) section of the NGINX Ingress Controller docs for more information on available command line arguments.
 
-1. Add the following Prometheus annotations NGINX Plus Ingress Controller Pod spec:
+1. Add the following Prometheus annotations NGINX Ingress Controller Pod spec:
 
    ```yaml
    prometheus.io/scrape: "true"
    prometheus.io/port: "<prometheus-metrics-listen-port>"
    ```
 
-1. Add the resource name as a label to the NGINX Plus Ingress Controller Pod spec:
+1. Add the resource name as a label to the NGINX Ingress Controller Pod spec:
 
     - For *Deployment*:
 
       ```yaml
-      nsm.nginx.com/deployment: <name of NGINX Plus Ingress Controller Deployment>
+      nsm.nginx.com/deployment: <name of NGINX Ingress Controller Deployment>
       ```
 
     - For *DaemonSet*:
 
       ```yaml
-      nsm.nginx.com/daemonset: <name of NGINX Plus Ingress Controller DaemonSet>
+      nsm.nginx.com/daemonset: <name of NGINX Ingress Controller DaemonSet>
       ```
 
-    This allows metrics scraped from NGINX Plus Ingress Controller Pods to be associated with the resource that created the Pods.
+    This allows metrics scraped from NGINX Ingress Controller Pods to be associated with the resource that created the Pods.
 
 ### View the metrics in Prometheus
 
-The NGINX Service Mesh uses the Pod's container name setting to identify the NGINX Plus Ingress Controller metrics that should be consumed by the Prometheus server.
+The NGINX Service Mesh uses the Pod's container name setting to identify the NGINX Ingress Controller metrics that should be consumed by the Prometheus server.
 The Prometheus job targets all Pods that have the container name `nginx-plus-ingress`.
 
-Add the `nginx-plus-ingress` scrape config to your Prometheus configuration and consult
```

Comment:
We probably want two sections for each of the two types of scrape configs. I see you added an OSS scrape config.